src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/weapons/SeismicAxe.java

```diff
@@ -150,7 +151,7 @@ private void pushEntity(Player p, Entity entity) {
 
     private @Nonnull Block findGround(@Nonnull Block b) {
         if (b.getType() == Material.AIR) {
-            for (int y = 0; y < b.getY(); y++) {
+            for (int y = WorldUtils.getMinHeight(b.getWorld()); y < b.getY(); y++) {
```

Comment:
0 is correct here by the way. These are *relative* coordinates, not absolutes.

It's actually the condition `y < b.getY()` which is responsible for the world limit.
I suppose this should rather be "while" `b.getY() - y > minWorldSize`
```suggestion
            for (int y = 0; y < b.getY(); y++) {
```

Reply:
This ^